src/coin/skycoin/models/wallet.go

```diff
@@ -0,0 +1,273 @@
+package models
+
+import (
+	"github.com/fibercrypto/FiberCryptoWallet/src/core"
+	"github.com/fibercrypto/FiberCryptoWallet/src/util"
+	"github.com/skycoin/skycoin/src/api"
+	"github.com/skycoin/skycoin/src/readable"
+)
+
+const (
+	Sky      = "Sky"
+	CoinHour = "CoinHour"
+)
+
+type SkycoinWalletIterator struct { //Implements WalletIterator interface
+	current int
+	wallets []Wallet
+}
+
+func (it *SkycoinWalletIterator) Value() core.Wallet {
+	return it.wallets[it.current]
+}
+
+func (it *SkycoinWalletIterator) Next() bool {
+	if it.HasNext() {
+		it.current++
+		return true
+	}
+	return false
+}
+
+func (it *SkycoinWalletIterator) HasNext() bool {
+	if (it.current + 1) >= len(it.wallets) {
+		return false
+	}
+	return true
+}
+
+func NewSkycoinWalletIterator(wallets []Wallet) *SkycoinWalletIterator {
+	return &SkycoinWalletIterator{wallets: wallets, current: -1}
+}
+
+type WalletService struct { //Implements WalletStorage and WalletSet interfaces
+}
+
+func (wltSrv *WalletService) ListWallets() core.WalletIterator {
```

Comment:
`WalletService` => `SkycoinRemoteWallet` . This is to differentiate this from the other implementation based on Skycoin core classes .